Guided Investigation - Fusion Incident Triage.ipynb

```diff
@@ -0,0 +1,1764 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ws_name = "fusionsyntheticeus02"
</pre><p>You probably want to set this to something like "Default"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
nice catch, fixed

>